src/datamodel/builder/Payload.cpp

```diff
@@ -10,3 +10,100 @@
 // SPDX-License-Identifier: Apache-2.0
 
 #include "up-cpp/datamodel/builder/Payload.h"
+namespace uprotocol::datamodel::builder {
+
+// Byte array constructor
+Payload::Payload(const std::vector<uint8_t>& value_bytes,
+                 v1::UPayloadFormat format) {
+	if (format < v1::UPayloadFormat::UPAYLOAD_FORMAT_UNSPECIFIED ||
```

Comment:
Please surround each logical pair with parenthesis.

Reply:
Done. Also as per david suggestion, I made the logical check in a common function.